__tests__/endpoints.test.js

```diff
@@ -127,4 +127,54 @@ describe("/api/articles", () => {
         })
     })
     
+})
+
+describe("/api/articles/:article_id/comments", () => {
+    test("200: returns with and array of comments with correct keys for article with specific ID", () => {
+        return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({body}) => {
+            const comments = body[0]
+            const size = Object.keys(comments[0]).length;
+            expect(size).toBe(6)
+            comments.forEach((comment) => {
+                expect(comment).toMatchObject({
+                    comment_id: expect.any(Number),
+                    article_id: expect.any(Number),
+                    body: expect.any(String),
+                    created_at: expect.any(String),
+                    votes: expect.any(Number),
+                    author: expect.any(String),
+                })
+            })
+        })
+    })
+    test("articles appear in descending order", () => {
+        return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({body}) => {
+            const comments = body[0]
+            expect(comments).toBeSortedBy('comment_id', {
+                descending: true,
+              });
+        })
+    })
+    test("404: check error message is correct when article number doesn't exist", () =>{
+        return request(app)
+        .get("/api/articles/33/comments ")
+        .expect(404)
+        .then(({body}) => {
+            expect(body.msg).toBe("Not Found")
+        });
+    });
+    test("400: Bad request when given invalid article ID", () => {
+        return request(app)
+        .get("/api/articles/banana/comments")
+        .expect(400)
+        .then(({body}) => {
+            expect(body.msg).toBe("Bad Request")
+        })
+    })
```

Comment:
Really good tests so far, but one final test that should be added is when given given a valid article id which does not have any associated comments. What should we expect in this case?